modules/azure/function_app_linux_managed_identity/main.tf

```diff
@@ -41,12 +41,15 @@ locals {
 # Function App
 
 resource "azurerm_linux_function_app" "function_app" {
-  name                        = var.name
-  location                    = var.location
-  resource_group_name         = var.resource_group_name
-  service_plan_id             = var.service_plan_id
-  storage_account_name        = var.storage_account_name
-  storage_account_access_key  = var.storage_account_access_key
+  name                 = var.name
+  location             = var.location
+  resource_group_name  = var.resource_group_name
+  service_plan_id      = var.service_plan_id
+  storage_account_name = var.storage_account_name
+
+  storage_account_access_key    = (var.use_managed_identity == null || var.use_managed_identity == false) ? var.storage_account_access_key : null
```

Comment:
`use_managed_identity` can never be null, because the default is set to true.

Meaning if I have a existing project using this with the access key and no `use_managed_identity` set then this will trigger change breaking everything.

Please change the default of `use_managed_identity` to `null`.